garden_ai/client.py

```diff
@@ -436,18 +412,14 @@ def get_local_garden(self, doi: str) -> Garden:
             exception if not found.
 
         """
-        registered = local_data.get_local_garden_by_doi(doi)
+        garden = local_data.get_local_garden_by_doi(doi)
 
-        if registered is None:
+        if garden is None:
             raise GardenNotFoundException(
                 f"Could not find any Gardens with identifier {doi}."
             )
-        self._set_up_mlflow_env()
-        registered._env_vars = {
-            "MLFLOW_TRACKING_TOKEN": os.environ["MLFLOW_TRACKING_TOKEN"],
-            "MLFLOW_TRACKING_URI": os.environ["MLFLOW_TRACKING_URI"],
-        }
-        return registered
+        self._generate_presigned_urls_for_garden(garden)
```

Comment:
Note that we grab the URLs needed for all the pipelines in a garden when we make a garden from the client. And we make all the requests in sequence, not concurrently. So if someone makes a big garden with 10+ models this will be pokey. For now I just wanted to go for correct and easy to understand, but there are a lot of ways we could improve this and it could be a nice task for a student or outside collaborator.